spec/system/authentication/user_logs_in_with_github_spec.rb

```diff
@@ -49,6 +49,10 @@
     end
 
     context "when using invalid credentials" do
+      let(:params) do
+        '{"callback_url"=>"http://localhost:3000/users/auth/github/callback", "state"=>"navbar_basic"}'
```

Comment:
Hm, iirc, there is a way in rspec to set a host and then use it to build a path? Not blocking, but we might want to consider that instead of explicitly hardcoding `localhost:3000` into our tests 🤔

Reply:
I honestly don't know :D

Reply:
Also, `config/test.rb` is setup like this:

```ruby
Rails.application.routes.default_url_options = { host: "test.host" }
```

which apparently is being ignored :-D

Reply:
So the `default_url_options` are correctly set to `{:host=>"test.host"}` when I inspect them in the spec. 

But this is a system test, which means it run through Capybara so you're potentially having a problem with `Capybara.default_host` or something.